src/source-dpdk.c

```diff
@@ -104,6 +141,7 @@ typedef struct DPDKThreadVars_ {
     TmSlot *slot;
     LiveDevice *livedev;
     ChecksumValidationMode checksum_mode;
+    int16_t intr_en;
```

Comment:
if it looks like a bool, quacks like a bool...

Reply:
booled!